binding.cc

```diff
@@ -223,12 +223,14 @@ int worker_send(worker* w, const char* msg) {
   return 0;
 }
 
-worker* worker_new(worker_recv_cb cb, void* data) {
+void v8_init () {
```

Comment:
I guess here is not following the author's style, remove the blank before the `()`?